tsconfig.json

```diff
@@ -16,7 +16,12 @@
     "noEmit": true,
     "jsx": "react-jsx"
   },
-  "include": ["./src"],
+  "include": [
```

Comment:
include `types-for-adobe` in root, so when importing extendscript function types, ts can find there type defs